.idea/gradle.xml

```diff
@@ -3,9 +3,9 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="distributionType" value="DEFAULT_WRAPPED" />
```

Comment:
You should always use the default wrapper (i.e `DEFAULT_WRAPPED` ) so that the Gradle system can manage the build versions.